### PR TITLE
sb 8.1.10

### PR DESCRIPTION
**Pull request by abernier**
### Checklist

- [ ] Documentation updated ([example](https://github.com/pmndrs/drei/blob/master/README.md#example))
- [ ] Storybook entry added ([example](https://github.com/pmndrs/drei/blob/master/.storybook/stories/Example.stories.tsx))
- [ ] Ready to be merged

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -8,14 +8,6 @@ const preview: Preview = {
     layout: 'fullscreen',
   },
 
-  decorators: [
-    (Story) => (
-      <React.Suspense fallback={null}>
-        <Story />
-      </React.Suspense>
```
Comment (abernier) [inline .storybook/preview.tsx:15]: @CodyJasonBennett | @drcmda: I think this is not required anymore since r3f now has this

File: package.json [hunk 2]
```diff
@@ -137,10 +137,10 @@
     "rollup-plugin-terser": "^7.0.2",
     "semantic-release": "^21.0.6",
     "serve": "^14.2.0",
-    "storybook": "^8.1.5",
+    "storybook": "^8.1.10",
     "three": "^0.151.0",
     "ts-node": "^10.9.1",
-    "typescript": "^4.7.4",
```
Comment (abernier) [inline package.json:143]: bumping TS too

File: .storybook/preview.tsx (hunk 1 above)
Comment (CodyJasonBennett) [reply .storybook/preview.tsx:15]: I believe this was here since Storybook would crash prior if something suspended into its component tree.

Comment (CodyJasonBennett) [reply .storybook/preview.tsx:15]: Let's try removing it.

**Review by abernier — COMMENTED** [review]
lgtm

**Pull request merged**